src/main/java/io/milvus/param/collection/CreateCollectionParam.java

```diff
@@ -58,9 +58,9 @@ public String getDescription() {
 
 
     public static final class Builder {
-        private String collectionName;
-        private int shardsNum;
-        private String description;
+        private String collectionName = "";
```

Comment:
The default collectionName set to "" is not a good idea, user may create a collectionName with empty string. 
Better to throw exception if collection name is null when call build

Reply:
make sense